pkg/operator/controller/utils/test/vars.go

```diff
@@ -21,8 +21,9 @@ import (
 )
 
 const (
-	TestNamespace     = ""
-	OperatorNamespace = "node-observability-operator"
+	TestNamespace            = ""
+	OperatorNamespace        = "node-observability-operator"
+	KubeletCASourceNamespace = "openshift-config-managed"
```

Comment:
Do you prefer to redefine the namespaces here, as opposed to reusing from names.go?

Reply:
We are using them for tests specifically, I am not really sure if I should use it from names.go